common/pom.xml

```diff
@@ -128,6 +128,11 @@
             <version>${spring.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+            <version>4.0.3.RELEASE</version>
+        </dependency>
```

Comment:
Why do we need this?  The version doesn't match how the other spring dependencies are being injected.  Maybe a spring boot dependency would be better if it's required?